content/support.rst

```diff
@@ -0,0 +1,193 @@
+
+
+Support Case Management
+=======================
+
+
+Working on a support case
+-------------------------
+
+Support cases can be received either through the online contact form,
+direct email or phone call.
+
+Email and support forms are the preferred methods.
+When we have a phone call, we request the customer at the beginning to do a
+follow up over email and confirm in writing.
+
+Support can be requested by partners or customers.
+
+When a support case is started, obtain and collate as much information as
+possible.
+This will help in potential follow-up with the rest of the Support team.
+
+
+Troubleshooting a Support Case
+------------------------------
+
+The `troubleshooting theory from CompTIA <http://certmag.com/guide-troubleshooting-theory-comptia-perspective/>`_ is a good overview when
+troubleshooting a support case:
+
+1. Identify the problem
+2. Establish a theory of probable cause (question the obvious)
+3. Test the theory to determine cause
+4. Establish a plan
+5. Determine system status
+6. Make a record
+
+
+Initial Assessment
+------------------
+
+When a support case has started, it should receive a response that it has
+been received and a case open within the same or next business day.
+
+The case will be allocated to an Owner, who should do the initial assessment:
+
+* Verify all details logged against the call, checking what additional
+  information is required;
+
+* Make initial contact with the originator, acknowledge that we have receive
+  the initial case;
+
+* Verify understanding of initial information provided;
+
+* Ask for additional information if required and/or provided initial diagnosis;
+
+* If more time is needed before providing an answer, inform about the
+  approximate time.
+
+If a customer asks for multiple support questions in a single email, it is
+better to respond in multiple emails.
+
+
+Managing a Support Case
+-----------------------
+
+The helpdesk allows you to keep track of which support cases still need a
+response.
+Once a response is made, close the helpdesk ticket as to not accumulate
+on the helpdesk dashboard.
+The helpdesk ticket will re-open should there be any further responses made.
+
+If the support case is waiting on a developer (in the case of a defect
+for example), a Trac ticket should be created with the helpdesk link and the
+defect report in the description.
+
+
+Escalating a Support Case due to defects
+----------------------------------------
+
+If the support case leads to finding a defect, a new Trac ticket should be
+created with details of the customer and link to the helpdesk conversation.
+
+For the defect, create a Trac ticket with priority High and notify the
+customer of the Trac ticket ID so that they can follow up with Support on the
+issue.
+
+
+Holding a screenshare session
+-----------------------------
+
+GoToMeeting can be used to conduct a screenshare or meeting session with the
+customer if the issue is best resolved via screenshare.
```

Comment:
I think that here we should add some best practices.

Like forcing users to use a phone, as the VOIP is bad.

Ask users to use private phones and not the shared loud phone found in the conference room.

Some information about planning the meeting. Send calendar invites. Add info to gotomeeting URL

Reply:
The Sales PR https://github.com/chevah/styleguide/pull/63/files has more details about GoToMeeting with the tips.

I was planning to update this page if/when the Sales PR is merged (to avoid double up) and then refer to that page for tips.